pkg/gofr/http/errors.go

```diff
@@ -0,0 +1,76 @@
+package http
+
+import (
+	"fmt"
+	"net/http"
+	"strings"
+)
+
+// ErrorEntityNotFound represents an error for when an entity is not found in the system.
+type ErrorEntityNotFound struct {
```

Comment:
We need to see if Error will be prefix or suffix and decide on one naming convention as DBErr and ErrorEntityNotFound are not consistent

Reply:
we will use `Error` as prefix only as suggested by naming convention.